### PR TITLE
Transfer repo to rel-eng

### PR DESCRIPTION
**Pull request by i-like-robots**
- Replaces `n-gage` with `rel-engage`
- Applies all Prettier and linting rules as specified by the above
- Updates the Circle config to match the publishing flow used by Reliability Engineering

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# generator: n-circle2-cli
-# template: component
+version: 2.1
+
+orbs:
+  ft-snyk-orb: financial-times/ft-snyk-orb@0
```
Comment (nl-ria) [inline .circleci/config.yml:4]: Might wanna use [Change API orb](https://github.com/Financial-Times/change-api-orb) too to notify on usual team channels upon releases

File: Makefile [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+# If, by the end of parsing your `Makefile`, `make` finds that any files
+# referenced with `-include` don't exist or are out of date, it will run any
+# tasks it finds that match the missing file. So if n-gage *is* installed
+# it will just be included; if not, it will look for a task to run
+-include node_modules/@financial-times/rel-engage/index.mk
+
+# End generated by rel-engage
+# ---------------------------
+
+PROJECT_NAME=athloi
```
Comment (wheresrhys) [inline Makefile:22]: probably don't need this as PROJECT_NAME is really only needed for referencing a folder in vault... though does no harm I gues

File: .circleci/config.yml [hunk 3]
```diff
@@ -99,37 +70,35 @@ jobs:
...
-          command: .circleci/shared-helpers/helper-npm-version-and-publish-public
+          name: npm publish
+          command: |
+            echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" > ${HOME}/.npmrc
+            make npm-publish
 
 workflows:
 
-  version: 2
+  version: 2.1
```
Comment (nl-ria) [inline .circleci/config.yml:86]: This line is redundant I think from version 2.1 onwards

**Comment by lucas42** [comment]
Is it worth adding a `CODEOWNERS` file too?

File: .circleci/config.yml (hunk 1 above)
Comment (i-like-robots) [reply .circleci/config.yml:4]: Sure, is this something other components do on release?

Comment (nl-ria) [reply .circleci/config.yml:4]: ah right, this is a package. We do not currently support logging release for packages via Change API, so maybe not 😅

**Comment by i-like-robots** [comment]
Codeowners is present and correct in `master` already 👍

**Pull request merged**